docs/mkdocs.yml

```diff
@@ -182,6 +182,7 @@ nav:
       - bucket:
         - bucket add: 'cmd/planner/bucket/bucket-add.md'
         - bucket list: 'cmd/planner/bucket/bucket-list.md'
+        - bucket list: 'cmd/planner/bucket/bucket-get.md'
```

Comment:
`bucket list` should be `bucket get` and should be placed above `bucket list` to maintain alphabetical order.

Reply:
done